src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -1960,6 +1972,27 @@ - (nullable id)accessibilityParent
 
 @end
 
+
+/*
+ * Implementation of the NSAccessibilityButton protocol
+ */
+@implementation ButtonAccessibility
+- (nullable NSString *)accessibilityLabel
+{
+    return [self accessibilityTitleAttribute];
+}
+
+- (BOOL)accessibilityPerformPress
+{
+    AWT_ASSERT_APPKIT_THREAD;
+
+    JNIEnv* env = [ThreadUtilities getJNIEnv];
+    JNFCallStaticVoidMethod(env, jm_doAccessibleAction, [self axContextWithEnv:(env)], 0, fComponent);
```

Comment:
Are you sure that the `jm_doAccessibleAction` is always initialized here?

Reply:
Yes.

Reply:
I guess it is became broken when the perform() will be removed. Can this method use its own variable?